.buildkite/scripts/lib/manifest.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
```

Comment:
Might be good to write a paragraph here about what the manifest is, why we have it, what it's used for, etc.

Reply:
done